Source/WebInspectorUI/UserInterface/Views/Popover.js

```diff
@@ -439,10 +441,10 @@ WI.Popover = class Popover extends WI.Object
         bounds = bounds.inset(WI.Popover.ShadowEdgeInsets);
         let computedStyle = window.getComputedStyle(this._element, null);
 
-        let context = WI.Popover._getCanvasContext(scaledWidth, scaledHeight);
+        let context = WI.Popover._getCanvasContext(width, height, scaleFactor);
         this._element.appendChild(context.canvas);
 
-        context.clearRect(0, 0, scaledWidth, scaledHeight);
+        context.clearRect(0, 0, width * scaleFactor, height * scaleFactor);
```

Comment:
NIT: and instead of using `window.devicePixelRatio` here we could just use `context.canvas.width`/`context.canvas.height`?